Monika After Story/game/pong.rpy

```diff
@@ -430,29 +603,503 @@ init -1 python in mas_pong:
+                m 1hub "I believe in you~"
+            else:
+                m 2eua "Concentrate, [player]." 
+                m 3hua "I'm sure you will win soon."
+                m 3hub "Just keep trying!"
+
+            $ pong_monika_last_response_id = PONG_MONIKA_RESPONSE_WIN_EASY_GAME
+
+
+        #On medium difficulty
+        elif pong_difficulty_before <= 10:
+            if pong_monika_last_response_id == PONG_MONIKA_RESPONSE_WIN_MEDIUM_GAME:
+                m 1hub "I win another round~"
+            else:
+                if loss_streak_counter > 1:
+                    m 3hub "Looks like I won again."
+                else:
+                    m 2hub "Looks like I won."
+
+            $ pong_monika_last_response_id = PONG_MONIKA_RESPONSE_WIN_MEDIUM_GAME           
+
+
+        #On hard difficulty
+        elif pong_difficulty_before <= 15:
+            if pong_monika_last_response_id == PONG_MONIKA_RESPONSE_WIN_HARD_GAME:
+                m 1hub "Ahaha!"
+                m 3tsb "Am I playing too good for you?"
+                m 3tub "I'm just kidding, [player]."
+                m 4hub "You're pretty good yourself!"
+            else:
+                if loss_streak_counter > 1:
+                    m 1huu "I win again~!"
+                else:
+                    m 1huu "I win~!"
+
+            $ pong_monika_last_response_id = PONG_MONIKA_RESPONSE_WIN_HARD_GAME 
+
+
+        #On expert difficulty
+        elif pong_difficulty_before <= 20:
+            if pong_monika_last_response_id == PONG_MONIKA_RESPONSE_WIN_EXPERT_GAME:
+                show monika 5eub at t11 zorder MAS_MONIKA_Z with dissolve
+
+                m 5hub "It feels good to win!"
+                m 5eua "Don't worry, I'm sure you will win again soon~"
+            else:
+                if loss_streak_counter > 1:
+                    m 2eub "I win another round!"
+                else:
+                    m 2eub "I win this round!"
+
+            $ pong_monika_last_response_id = PONG_MONIKA_RESPONSE_WIN_EXPERT_GAME
+
+
+        #On extreme difficulty
+        else:
+            if pong_monika_last_response_id == PONG_MONIKA_RESPONSE_WIN_EXTREME_GAME:
+                m 2duu "Not bad, [player]."
+                m 4esb "I gave it everything I had, so don't feel too bad for losing from time to time."
+            else:
+                m 2hub "This time it's my win!"
+                m 2efu "Keep up, [player]!"
+
+            $ pong_monika_last_response_id = PONG_MONIKA_RESPONSE_WIN_EXTREME_GAME
+
     return
 
-# end dialogue shown when monika is the pong winner
-label mas_pong_dlg_winner_end:
-    m 4tku "I can't really get excited for a game this simple..."
-    m 1eua "At least we can still hang out with each other."
-    m 1hub "Ahaha!"
-    m 1eua "Thanks for letting me win, [player]."
-    m 1tku "Only elementary schoolers seriously lose at Pong, right?"
-    m 1hua "Ehehe~"
+
+
+# dialogtue shown right when monika loses
```

Comment:
Maybe you'll want to fix this comment at some point.

Reply:
Does this one pop up the first time you play?

Reply:
It is also worth thinking about to change the "you are awesome at pong" compliment.

Reply:
Oh you meant the comment in the code, not monika's comment.